### PR TITLE
Add more info to the README, change name 'Misq' -> 'Haveno' everywhere, remove Github templates etc.

### PR DESCRIPTION
**Pull request by erciccione**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,55 @@
...
 
 ## Running a local test network
 
 1. Download [Monero CLI](https://www.getmonero.org/downloads/) for your system
-2. Sync stagenet: `./monerod --stagenet`, or alternatively, start a local stagenet network:
+2. Sync stagenet: `./monerod --stagenet --rpc-login superuser:abctesting123 --confirm-external-bind`, or alternatively, start a local stagenet network:
     * `./monerod --stagenet --no-igd --hide-my-port --data-dir node1 --p2p-bind-ip 127.0.0.1 --p2p-bind-port 48080 --rpc-bind-port 48081 --zmq-rpc-bind-port 48082 --add-exclusive-node 127.0.0.1:38080 --rpc-login superuser:abctesting123 --rpc-access-control-origins http://localhost:8080`<br>
     * `./monerod --stagenet --no-igd --hide-my-port --data-dir node2 --p2p-bind-ip 127.0.0.1 --rpc-bind-ip 0.0.0.0 --confirm-external-bind --add-exclusive-node 127.0.0.1:48080 --rpc-login superuser:abctesting123 --rpc-access-control-origins http://localhost:8080`
-3. Download and install [Bitcoin-Qt](https://bitcoin.org/en/download)
+3. Download and install [Bitcoin-Qt](https://bitcoin.org/en/download) (needed if you want to test a XMR <-> BTC exchange)
```
Comment (woodser) [inline README.md:53]: Pretty sure this is currently a dependency for the application to function correctly.

File: README.md [hunk 2]
```diff
@@ -2,15 +2,55 @@
...
+
+At the moment Haveno is simply a Proof Of Concept. It's already possible to initiate a trade, but the project still needs a lot of work before being available for public use.
+
+These are the main priorities for the near future:
+
+- We need a logo. Right now it's a random stock image :)
+- The User Interface is basically still Bisq. The entire interface needs to be completely reworked and adapted for Monero as base currency
+- Create documentation for developers
+- Cleanup the repository from Bisq-specific content
+- Organzie what Bisq calls "Arbitrators"
```
Comment (woodser) [inline README.md:28]: Organize

File: README.md [hunk 3]
```diff
@@ -2,15 +2,55 @@
...
+We have our own Matrix server. Registrations are not open at the moment, but the rooms are public and can be joined from any matrix client (like Element):
+
+- General discussions: **Haveno** (`#haveno:haveno.network`) relayed on Freenode (`#haveno`)
+- Development discussions: **Haveno Development** (`haveno-dev:matrix.network`) relayed on Freenode (`#haveno-dev`)
+
+You can find the rooms by adding the 
+
+## Status of the project
+
+At the moment Haveno is simply a Proof Of Concept. It's already possible to initiate a trade, but the project still needs a lot of work before being available for public use.
```
Comment (woodser) [inline README.md:20]: Nit: "proof of concept" or "Proof of Concept"

**Review by woodser — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (erciccione) [reply README.md]: Removed.

**Comment by erciccione** [comment]
Rebased.

**Pull request merged**